index.html

```diff
@@ -312,14 +312,16 @@ <h3>
 listed here.
           </p>
           <ul>
+            <li>Test suites for all normative deliverables</li>
             <li>Presentation Request Data Model</li>
             <li>Storage and Sharing of Verifiable Credentials</li>
             <li>Privacy Guidance for Verifiable Credentials</li>
+            <li>Extensions for binding multilingual resources for localized user interfaces.</li>
             <li>A Developer Guide consisting of one or more notes related to general implementation guidance and best practices for working with VCs, including but not limited to:
               <ul>
                 <li>One or more HTTP protocol definitions for Verifiable Credential Exchange (such as the <a href="https://w3c-ccg.github.io/vc-api/">VC-API</a>)</li>
                 <li>Guidance on Verifiable Credential Exchange over OpenID Connect</li>
-                <li>Verifiable Credential Exchange over [GNAP](https://datatracker.ietf.org/doc/html/draft-ietf-gnap-core-protocol)</li>
+                <li>Verifiable Credential Exchange over <a href="https://datatracker.ietf.org/doc/html/draft-ietf-gnap-core-protocol">GNAP</a></li>
                 <li>Other protocols as time, attention, and resources permit.</li>
```

Comment:
For consistency in this bullet list
```suggestion
                <li>Other protocols as time, attention, and resources permit</li>
```